### ISSUE TITLE
How to change the value of the field?

### ISSUE THREAD

**Issue by xvrh** [cedvdb/phone_form_field#6]
Hi! Thanks for this package.
`PhoneFormField` exposes `initialValue` but doesn't expose the `TextEditingController`. 
How to change the value of the field after creation (ie. to implement a picker button from a contact list)?

Thanks!

**Comment by cedvdb** [comment]
Hi! I just woke up so I could be wrong, but let me rephrase your use case as I'm not totally sure I understand it:

 - an user picks a contact on a screen A
 - The contact is shown on a screen B
 - on screen B the phone field is also shown with the contact phone number 
 
 Is that correct ?

**Comment by xvrh** [comment]
My use case is: I want to have a PhoneFormField that the user can edit. And also add a button "Pick from list" to open a dialog box to allow directly pick a phone number.
When the user pick the phone from the list, override the current field value.

I was able to workaround it by changing the key to re-create the state and specifying a new `initialValue`

**Comment by cedvdb** [comment]
new controller exposed in next release
